<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNetTool</class>
 <widget class="QWidget" name="frmNetTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTcpServer">
      <attribute name="title">
       <string>TCP服务器</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="txtDataTcpServer"/>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QFrame" name="frmTcpServer">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ckHexReceiveTcpServer">
            <property name="text">
             <string>16进制接收</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckHexSendTcpServer">
            <property name="text">
             <string>16进制发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckDebugTcpServer">
            <property name="text">
             <string>模拟设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckAutoSendTcpServer">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxIntervalTcpServer"/>
          </item>
          <item>
           <widget class="QLabel" name="labTcpListenPort">
            <property name="text">
             <string>监听端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTcpListenPort"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnTcpListen">
            <property name="text">
             <string>监听</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearTcpServer">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labTcpClientCount">
            <property name="text">
             <string>已连接客户端共 0 个</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listTcpClient"/>
          </item>
          <item>
           <widget class="QCheckBox" name="ckAllTcpServer">
            <property name="text">
             <string>对全部客户端发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="layTcpServer">
         <item>
          <widget class="QComboBox" name="cboxSendTcpServer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendTcpServer">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTcpClient">
      <attribute name="title">
       <string>TCP客户端</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="txtDataTcpClient"/>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="layTcpClient">
         <item>
          <widget class="QComboBox" name="cboxSendTcpClient">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendTcpClient">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QFrame" name="frmTcpClient">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ckHexReceiveTcpClient">
            <property name="text">
             <string>16进制接收</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckHexSendTcpClient">
            <property name="text">
             <string>16进制发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckDebugTcpClient">
            <property name="text">
             <string>模拟设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckAutoSendTcpClient">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxIntervalTcpClient"/>
          </item>
          <item>
           <widget class="QLabel" name="labTcpServerIP">
            <property name="text">
             <string>服务器IP地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTcpServerIP"/>
          </item>
          <item>
           <widget class="QLabel" name="labTcpServerPort">
            <property name="text">
             <string>服务器端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTcpServerPort"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnTcpConnect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearTcpClient">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUdpServer">
      <attribute name="title">
       <string>UDP服务器</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="txtDataUdpServer"/>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QFrame" name="frmUdpServer">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="ckHexReceiveUdpServer">
            <property name="text">
             <string>16进制接收</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckHexSendUdpServer">
            <property name="text">
             <string>16进制发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckDebugUdpServer">
            <property name="text">
             <string>模拟设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckAutoSendUdpServer">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxIntervalUdpServer"/>
          </item>
          <item>
           <widget class="QLabel" name="labUdpClientIP">
            <property name="text">
             <string>远程IP地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUdpClientIP"/>
          </item>
          <item>
           <widget class="QLabel" name="labUdpClientPort">
            <property name="text">
             <string>远程端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUdpClientPort"/>
          </item>
          <item>
           <widget class="QLabel" name="labUdpListenPort">
            <property name="text">
             <string>监听端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUdpListenPort"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnUdpListen">
            <property name="text">
             <string>监听</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearUdpServer">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="layUdpServer">
         <item>
          <widget class="QComboBox" name="cboxSendUdpServer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendUdpServer">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
